cost-analyzer/values.yaml

```diff
@@ -787,3 +787,4 @@ awsstore:
 #    secretname: productkeysecret # create a secret out of a file named productkey.json of format { "key": "kc-b1325234" }
 #    mountPath: "/some/custom/path/productkey.json" # (use instead of secretname) declare the path at which the product key file is mounted (eg. by a secrets provisioner). The file must be of format { "key": "kc-b1325234" }
 #  cloudIntegrationSecret: "cloud-integration"
+#  ingestPodUID: false # Enables using UIDs to uniquely ID pods. This requires either Kubecost's replicated KSM metrics, or KSM v2.1.0+. This may impact performance, and changes the default cost-model allocation behavior.
```

Comment:
Comment above I realized got stuck in a pending state... sorry about that. @kaelanspatel do we have more detail on this change?

cc @AdamStack18

Reply:
@dwbrown2, didn't see this last week.

I definitely thought about documenting this in more depth, but there's a very specific edge case where this is the solution. I talked it over with some people and decided against just inserting detailed notes (like you say). Ofc I'm not particularly happy with this vague comment you see here.

It seems like we have a lot of "trivia" about the app which we just expect people to ask about because it doesn't fit anywhere in the technical docs (e.g. What does this flag do? How is kubecost calculating this edge case? Is X an important warning or can I ignore it?)

Maybe we could have a set of clarification pages that we can add additional documentation to if we need to (like helm-chart-variables.md, edge-cases.md)?

Reply:
I think you make great points... what do you think about this proposal: 

Most flags have a short descriptions, e.g. 1-liner, directly in values. 
Then, those that require more detail (most?) have a "Learn more" or "Read more" link. 
Those links point to either a helm-chart-vars page or individual pages.